smart_card_connector_app/src/background.js

```diff
@@ -44,37 +46,61 @@ const JS_LOGS_HANDLER_MESSAGE_TYPE = 'js_logs_handler';
  */
 var logger = GSC.Logging.getScopedLogger('ConnectorApp.BackgroundMain');
 
-// Used to forward logs collected on the JS side to the NaCl module's stdout, in
-// order to simplify accessing them in some configurations.
-// Note that this object needs to be created as early as possible, in order to
-// not miss any important log.
-const logBufferForwarderToNaclModule = new GSC.LogBufferForwarder(
-    GSC.Logging.getLogBuffer(), JS_LOGS_HANDLER_MESSAGE_TYPE);
+if (GSC.ExecutableModule.TOOLCHAIN === GSC.ExecutableModule.Toolchain.PNACL) {
+  // Used to forward logs collected on the JS side to the NaCl module's stdout,
+  // in order to simplify accessing them in some configurations.
+  // Note that this object needs to be created as early as possible, in order to
+  // not miss any important log.
+  var logBufferForwarderToNaclModule = new GSC.LogBufferForwarder(
```

Comment:
Note: I wasn't able to use "let"/"const", since it'd be scoped to that "if". And declaring the variable with "let" before the "if" didn't work either - the compiler complained about assigning to an alias twice (which sounds like a compiler bug, but maybe I'm unaware about some assumption made in the Closure Compiler).